modules/components/AsyncConnect.js

```diff
@@ -44,9 +44,7 @@ export default class AsyncConnect extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    if (!this.context.store.getState().reduxAsyncConnect.loading) {
```

Comment:
This was firing on load actually because of `react-router-redux`.
Upgrading `react-router-redux` to ^4 resolved the issue.
Perhaps that should be specified as a peer dependency in this library?